code/client/munkilib/munkirepo/__init__.py

```diff
@@ -14,9 +15,16 @@ def import_plugins(dirpath=None):
     plugin_names = []
 
     if not dirpath:
-        # get the directory this __init__.py file is in
-        dirpath = os.path.dirname(os.path.abspath(__file__))
-
+        plugin_names = [
```

Comment:
This is super obtuse and non-obvious Please include comments about what this code is actually doing. Even the original code attempts to explain what it was doing with the obtuse code it was using.